config/routes.rb

```diff
@@ -1,3 +1,18 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  get '/new_post', to: 'posts#new'
+  post '/new_post', to: 'post#create'
+  get '/posts', to: 'posts#index'
+  root 'static_pages#home'
+  get '/help', to: 'static_pages#help'
+  get  '/signup',  to: 'users#new'
+  post '/signup',  to: 'users#create'
+  get    '/login',   to: 'sessions#new'
+  post   '/login',   to: 'sessions#create'
+  delete '/logout',  to: 'sessions#destroy'
+
+  resources :users
+  resources :posts
```

Comment:
Update that file with the routes needed for that project and corresponding to the requirements

Reply:
## Routes definitions updated:
```
  root 'static_pages#home'
  get '/help', to: 'static_pages#help'

  get '/new_post', to: 'posts#new'
  post '/new_post', to: 'post#create'
  get '/posts', to: 'posts#index'

  get    '/login',   to: 'sessions#new'
  post   '/login',   to: 'sessions#create'
  delete '/logout',  to: 'sessions#destroy'

  resources :users, only: %i[show index]
  resources :posts, only: %i[new create index]
```